frame/nomination-pools/src/lib.rs

```diff
@@ -1599,6 +1620,59 @@ pub mod pallet {
 			Ok(())
 		}
 
+		// TODO: Update Weight info
+		#[pallet::weight(
+			T::WeightInfo::bond_extra_transfer()
+			.max(T::WeightInfo::bond_extra_reward())
+		)]
+		pub fn root_bond_extra(
```

Comment:
would probably call this `force_bond_extra`, to avoid having the role explicitly mentioned.

Reply:
Yeah `force_bond_extra` is most ideal

Reply:
If we do as commented below, it should actually be called `bond_extra_other` to match the idiom of `chill_other` etc.

Reply:
In cases where pool members would want to pay the gas fees using another account, I can see this being a favorable change.

I think operators can also have a similar use case.